MarkerMetro.Unity.WinLegacyUnity/System/Security/Cryptography/EncryptionProvider.cs

```diff
@@ -137,11 +137,10 @@ public static string Decrypt(string cipherString, string key)
 
                 return strDecrypted;
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // MetroEventSource.Log.Error(ex.Message);
-                //throw;
-                return "";
+               System.Diagnostics.Debug.WriteLine(ex.Message);
+               throw new Exception("[EncryptionProvider] Error Decrypting a string");
```

Comment:
I think it should be changed regardless, there is no good reason to avoid throwing